### DISCUSSION TITLE
IP Range No Filter To security onion 2.3

### DISCUSSION THREAD

**Discussion by absemepror** [Security-Onion-Solutions/securityonion#3421] [Unsupported Versions] [answered]
Sorry, but in the new version in Elastic Search version 2.3, you can not enter the Range IP filter as a filter in the Add menu, and you must select the Range IP in the Query toolbar to monitor???
In version 16, this option is active and you can search the range of IP

**Comment by presianbg** [comment, marked as answer]
Hi,

Not sure if you are asking about the new SOC interface or about the Kibana.
In case you ask for Kibana:

![image](https://user-images.githubusercontent.com/2163486/110955356-2ec2b000-8352-11eb-95a0-ebcb7e968f06.png)

If your question is about the `Onion Query Language (OQL)`, I think its not possible to have wildcards or IP ranges there.

**Reply by presianbg** [reply]
@absemepror 
You can search like this too:
`(event.dataset:"rdp" AND (NOT (source.ip:(192.168.0.0/16 OR 10.0.0.0/8 OR 172.16.0.0/12))))`

**Reply by defensivedepth** [reply]
In this last release, we also added .keyword subfields for `destination.ip` and `source.ip` - so queries like this will work now: `source.ip.keyword: 192.168.*`
